src/test/groovy/graphql/TestUtil.groovy

```diff
@@ -2,32 +2,11 @@ package graphql
 
 import graphql.execution.MergedField
 import graphql.execution.MergedSelectionSet
-import graphql.language.Document
-import graphql.language.Field
-import graphql.language.NullValue
-import graphql.language.ObjectTypeDefinition
-import graphql.language.OperationDefinition
-import graphql.language.ScalarTypeDefinition
-import graphql.language.Type
+import graphql.introspection.Introspection.DirectiveLocation
+import graphql.language.*
 import graphql.parser.Parser
-import graphql.schema.Coercing
-import graphql.schema.DataFetcher
-import graphql.schema.GraphQLAppliedDirectiveArgument
-import graphql.schema.GraphQLAppliedDirective
-import graphql.schema.GraphQLArgument
-import graphql.schema.GraphQLDirective
-import graphql.schema.GraphQLInputType
-import graphql.schema.GraphQLObjectType
-import graphql.schema.GraphQLScalarType
-import graphql.schema.GraphQLSchema
-import graphql.schema.GraphQLType
-import graphql.schema.TypeResolver
-import graphql.schema.idl.RuntimeWiring
-import graphql.schema.idl.SchemaGenerator
-import graphql.schema.idl.SchemaParser
-import graphql.schema.idl.TestMockedWiringFactory
-import graphql.schema.idl.TypeRuntimeWiring
-import graphql.schema.idl.WiringFactory
+import graphql.schema.*
+import graphql.schema.idl.*
```

Comment:
Take out the `.*` imports - we dont like them.  You may have to change your IDE settings when working on the graphql code base